src/static/menu-win.json

```diff
@@ -320,6 +320,12 @@
                 {
                     "id": "NEW_ARTBOARD_FROM_TEMPLATE",
                     "submenu": []
+                },
+                {
+                    "separator": true
+                },
+                {
+                    "id":"NEW_VECTOR_MASK_FROM_SHAPE"
```

Comment:
Again, just want to confirm that you do not want a keyboard shortcut for this on Windows. If that is the case, why not?